### PR TITLE
Feature/flho146 obtain Step

### PR DESCRIPTION
**Pull request by bendiggle**
* Added the step to the routes config,
* Added the fields to the fields config,
* Added the translations to the fields pages and validation files. This includes deepTranslations,
* Added a template partial for the buy checkbox toggle,
* Added a line of sass for the partial removing a bottom margin,
* Added the acceptance tests for the obtain step,
* Added the page helper for obtain and storage,
* Added the helpers to the codecept config file.

### CODE REVIEW

**Comment by joefitter** [comment]
💩

**Comment by bendiggle** [comment]
@joefitter I'm going to take that as a LGTM!

**Comment by joefitter** [comment]
I want to make some changes to the deepTranslate which will affect this step, but, let's merge, bcoz agile

**Comment by joefitter** [comment]
@JoeChapman @gxxm

**Comment by bendiggle** [comment]
We're so agile!

File: apps/new-dealer/acceptance/pages/obtain.js [hunk 1]
```diff
@@ -1,5 +1,86 @@
 'use strict';
 
+const TRANSLATIONS = require('../../translations/en/default');
+const FIELDS = TRANSLATIONS.fields;
+const PAGES = TRANSLATIONS.pages;
```
Comment (JoeChapman) [inline apps/new-dealer/acceptance/pages/obtain.js]: Is this how we do? capitalise constants?

File: apps/new-dealer/fields/index.js [hunk 2]
```diff
@@ -3,7 +3,7 @@
 module.exports = {
   company: {
     mixin: 'radio-group',
-    validate: ['required'],
+    validate: 'required',
     className: ['form-group'],
```
Comment (JoeChapman) [inline apps/new-dealer/fields/index.js]: DO single className's need to be in array notation?

Comment (joefitter) [inline apps/new-dealer/fields/index.js]: either or

Comment (joefitter) [inline apps/new-dealer/fields/index.js]: string is nicer if only one value - we save 2 bytes ;)

Comment (bendiggle) [inline apps/new-dealer/fields/index.js]: Yep should be outside of the array. But doesn't link to this PR. Next one!

File: apps/new-dealer/acceptance/pages/obtain.js (hunk 1 above)
Comment (joefitter) [inline apps/new-dealer/acceptance/pages/obtain.js]: I like this pattern

Comment (bendiggle) [inline apps/new-dealer/acceptance/pages/obtain.js]: I like this pattern too. We've been doing it on the other forms so just been consistent here

**Pull request merged**